mxnet-bot/LabelBotFullFunctionality/README.md

```diff
@@ -77,3 +77,14 @@ As well, make sure to set the appropriate CNAME certificate from Certificate Man
 When wanting to update the stack using serverless deploy after initial launch, comment out
 in serverless.yml file the section regarding customDomain and plugins.
 ***Note:*** Confirm in each update that the basePath is set to /dev and point it to corret lambda under Custom Domain Names in API Gateway.
+
+
+#### 5. CloudWatch Log Access
+* In (deploy.sh) fill in the variables 
+    - dest_account="AWS_ACCOUNT_NUM" with the AWS account of the destination of the logs
+    - bot_account="AWS_ACCOUNT_NUM" with the AWS account of the bot account
+* Then, the destination log account proceeds to assume the role (to have view access of the logs)
+* After assuming the role proceed to view the logs by navigating to the appropriate section in the CloudWatch console
+   - [Label Logs](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logStream:group=/aws/lambda/LabelBot-dev-label)
```

Comment:
Who has the permission to access this link?

Reply:
Only those who are given access to view the label bot logs (or else its a dead link to everyone else)